[2022-10-04 12:05:22,942] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data manual__2022-10-04T12:05:17.491372+00:00 [queued]>
[2022-10-04 12:05:22,965] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data manual__2022-10-04T12:05:17.491372+00:00 [queued]>
[2022-10-04 12:05:22,966] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 12:05:22,967] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-04 12:05:22,968] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 12:05:22,985] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_raw_star_wars_people_data> on 2022-10-04 12:05:17.491372+00:00
[2022-10-04 12:05:22,991] {standard_task_runner.py:52} INFO - Started process 332 to run task
[2022-10-04 12:05:22,997] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_star_wars', 'get_raw_star_wars_people_data', 'manual__2022-10-04T12:05:17.491372+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/ingest_starwars_data.py', '--cfg-path', '/tmp/tmp4iav_mti', '--error-file', '/tmp/tmpn1ya9682']
[2022-10-04 12:05:22,999] {standard_task_runner.py:77} INFO - Job 26: Subtask get_raw_star_wars_people_data
[2022-10-04 12:05:23,106] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data manual__2022-10-04T12:05:17.491372+00:00 [running]> on host be1a5b89e146
[2022-10-04 12:05:23,202] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=ingest_star_wars
AIRFLOW_CTX_TASK_ID=get_raw_star_wars_people_data
AIRFLOW_CTX_EXECUTION_DATE=2022-10-04T12:05:17.491372+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-04T12:05:17.491372+00:00
[2022-10-04 12:05:34,085] {python.py:175} INFO - Done. Returned value was: None
[2022-10-04 12:05:34,096] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_star_wars, task_id=get_raw_star_wars_people_data, execution_date=20221004T120517, start_date=20221004T120522, end_date=20221004T120534
[2022-10-04 12:05:34,131] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-04 12:05:34,159] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
